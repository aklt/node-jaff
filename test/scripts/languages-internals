//begin js
var jsVar = 10;

//end
Language: {__language}
jsVar:    {jsVar}
#begin coffee
coffeeVar = """Foo
  is the thing bar

  all the time
"""
#end
Language: {__language}
coffeeVar: {coffeeVar}
#if false
#include if-elif-else-end.cmac
#include include1.js
#else
Potatoes
expand {__expand}
#end
#begin js
 // Now we are using javascript
#end

Langauges is {__language}
