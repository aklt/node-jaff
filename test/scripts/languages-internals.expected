Language: js
jsVar:    10
Language: coffee
coffeeVar: Foo
is the thing bar

all the time
Potatoes
expand {,}
